### ISSUE TITLE
Updating Node.js packages fails with Yarn v1.22.4

### ISSUE THREAD

**Issue by joschi** [r-darwish/topgrade#407]
**What did you expect to happen?**

Topgrade should successfully use Yarn (yarnpkg) to update installed Node.js packages.

**What actually happened?**

Topgrade fails because `yarn version` fails with a non-zero exit code.

https://github.com/r-darwish/topgrade/blob/cf34294a32d849a7ef248a4c71a51f1fc08df1ca/src/steps/node.rs#L56-L60

**Additional details:**
- Which operating system or Linux distribution are you using?
  - macOS 10.15.4
- How did you install Topgrade?
  - Homebrew: `brew install topgrade`
  - Topgrade 4.4.0

- If a specific step is failing, what happens when you run the command on your own? (Run `topgrade --dry-run` to see which commands Topgrade is running)
```
# /usr/local/bin/yarn version
yarn version v1.22.4
error Couldn't find a package.json file in "/Users/joschi"
info Visit https://yarnpkg.com/en/docs/cli/version for documentation about this command.
```

<details><pre>
# topgrade -v --only node
 2020-05-13T09:19:01.401Z DEBUG topgrade::config > Configuration at /Users/joschi/.config/topgrade.toml
 2020-05-13T09:19:01.401Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, commands: None, git_repos: None, predefined_git_repos: None, disable: None, remote_topgrades: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: None, yay_arguments: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, only: None, composer: None }
 2020-05-13T09:19:01.402Z DEBUG topgrade         > Version: 4.4.0
 2020-05-13T09:19:01.402Z DEBUG topgrade         > OS: x86_64-apple-darwin
 2020-05-13T09:19:01.402Z DEBUG topgrade         > Args { inner: ["topgrade", "-v", "--only", "node"] }
 2020-05-13T09:19:01.402Z DEBUG topgrade         > Binary path: Ok("/usr/local/bin/topgrade")
 2020-05-13T09:19:01.402Z DEBUG topgrade         > Self Update: false
 2020-05-13T09:19:01.403Z DEBUG topgrade::utils  > Detected "/usr/local/bin/git" as "git"
 2020-05-13T09:19:01.403Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"
 2020-05-13T09:19:01.403Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2020-05-13T09:19:01.403Z DEBUG topgrade::utils  > Cannot find "powershell"
 2020-05-13T09:19:01.403Z DEBUG topgrade::steps::git > /Users/joschi/.zshrc is a file. Checking /Users/joschi
 2020-05-13T09:19:01.403Z DEBUG topgrade::steps::git > Checking if /Users/joschi is a git repository
 2020-05-13T09:19:01.408Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: Kein Git-Repository (oder irgendeines der Elternverzeichnisse): .git\n" }
 2020-05-13T09:19:01.409Z DEBUG topgrade::steps::git > /Users/joschi/.tmux does not exists
 2020-05-13T09:19:01.409Z DEBUG topgrade::steps::git > /Users/joschi/.config/fish does not exists
 2020-05-13T09:19:01.409Z DEBUG topgrade::steps::git > /Users/joschi/Library/Preferences/openbox does not exists
 2020-05-13T09:19:01.409Z DEBUG topgrade::steps::git > /Users/joschi/Library/Preferences/bspwm does not exists
 2020-05-13T09:19:01.409Z DEBUG topgrade::steps::git > /Users/joschi/Library/Preferences/i3 does not exists
 2020-05-13T09:19:01.409Z DEBUG topgrade::steps::git > /Users/joschi/Library/Preferences/sway does not exists
 2020-05-13T09:19:01.409Z DEBUG topgrade::runner     > Step "NPM"
 2020-05-13T09:19:01.409Z DEBUG topgrade::utils      > Detected "/usr/local/bin/npm" as "npm"

―― 11:19:01 - Node Package Manager ―――――――――――――――――――――――――――――――――――――――――――――
 2020-05-13T09:19:01.409Z DEBUG topgrade::executor   > Running "/usr/local/bin/npm" "update" "-g"
 2020-05-13T09:19:02.722Z DEBUG topgrade::runner     > Step "yarn"
 2020-05-13T09:19:02.722Z DEBUG topgrade::utils      > Detected "/usr/local/bin/yarn" as "yarn"
 2020-05-13T09:19:02.931Z TRACE topgrade::executor   > Output of "/usr/local/bin/yarn" "version": Output { status: ExitStatus(ExitStatus(256)), stdout: "yarn version v1.22.4\ninfo Visit https://yarnpkg.com/en/docs/cli/version for documentation about this command.\n", stderr: "error Couldn\'t find a package.json file in \"/Users/joschi\"\n" }
 2020-05-13T09:19:02.931Z DEBUG topgrade::terminal   > Desktop notification: yarn failed

Retry? (y)es/(N)o/(s)hell

―― 11:19:51 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
NPM: OK
yarn: FAILED
</pre></details>

**Comment by r-darwish** [comment]
I think it's fixed by 9f8fbf6d6543320b5bebb39149eedbb624d0d3bc but I did not release a version yet. Can you please see if it reproduces on the master branch of Topgrade?

**Comment by joschi** [comment]
@r-darwish This seems to fix it. 👍

Should we close this issue as duplicate of #403?
